Gemfile

```diff
@@ -22,7 +22,7 @@ group :acceptance do
 end
 
 group :development do
-    gem 'github_changelog_generator', require: false
+    gem 'github_changelog_generator', '~> 1.15.0', require: false
```

Comment:
hey @david22swan , we (vox pupuli) had some problems with older versions of the gem, so we pinned to ~> 1.16.4. This was some time ago so I cannot remember the actual issues we faced.